keras_nlp/models/distil_bert/distil_bert_backbone_test.py

```diff
@@ -25,66 +25,60 @@
 
 class DistilBertTest(tf.test.TestCase, parameterized.TestCase):
     def setUp(self):
-        self.model = DistilBertBackbone(
-            vocabulary_size=1000,
+        self.backbone = DistilBertBackbone(
+            vocabulary_size=10,
             num_layers=2,
             num_heads=2,
-            hidden_dim=64,
-            intermediate_dim=128,
-            max_sequence_length=128,
+            hidden_dim=2,
+            intermediate_dim=4,
+            max_sequence_length=5,
             name="encoder",
         )
-        self.batch_size = 8
+
         self.input_batch = {
-            "token_ids": tf.ones(
-                (self.batch_size, self.model.max_sequence_length), dtype="int32"
-            ),
-            "padding_mask": tf.ones(
-                (self.batch_size, self.model.max_sequence_length), dtype="int32"
-            ),
+            "token_ids": tf.ones((2, 5), dtype="int32"),
+            "segment_ids": tf.ones((2, 5), dtype="int32"),
```

Comment:
distilbert does not take in `"segment_ids":`. Pleas make sure to run all the tests you are modifying, ideally locally before creating a PR.

`pytest keras_nlp/models/distil_bert` will run the basic tests, see our contributor guide for more details.